addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -433,6 +433,15 @@ X[]
                         contentAfter: `<p>[]&nbsp;def</p>`,
                     });
                 });
+                it('should remove invisible empty space at the start', async () => {
```

Comment:
Could you add this test in the commit about "delete forward invisible white spaces at start " please.
I think it add some context, coverage and make the fix a bit clearer.

```javascript               
 it('should remove invisible empty space at the start (2)', async () => {
    await testEditor(BasicEditor, {
        // The first 3 spaces are invisible : considered 
        // formating by the browser.
        // The &nbsp; is visible (space 1).
        // The last 3 spaces are consider as 1 visble space and 
        // two formating spaces (space 2).
        contentBefore: `<p>[]   &nbsp;   def</p>`,
        stepFunction: async editor => {
            await deleteForward(editor);
        },
        // Space 1 is deleted and space 2 should be transformed
        // to a &nbsp; to stay visible.
        contentAfter: `<p>[]&nbsp;def</p>`,
    });
});```